mobile/ios/device/ios-proxy-services.ts

```diff
@@ -266,7 +266,11 @@ export class InstallationProxyClient {
 	}
 
 	public closeSocket() {
-		return this.plistService.close();
+		if (this.plistService) {
```

Comment:
shouldn't this method close both sockets, if they exist, instead of prioritizing